shell/common/animator.cc

```diff
@@ -28,18 +29,7 @@ Animator::Animator(Delegate& delegate,
       last_frame_begin_time_(),
       last_frame_target_time_(),
       dart_frame_deadline_(0),
-#if FLUTTER_SHELL_ENABLE_METAL
-      layer_tree_pipeline_(fml::MakeRefCounted<LayerTreePipeline>(2)),
-#else   // FLUTTER_SHELL_ENABLE_METAL
-      // TODO(dnfield): We should remove this logic and set the pipeline depth
-      // back to 2 in this case. See
-      // https://github.com/flutter/engine/pull/9132 for discussion.
-      layer_tree_pipeline_(fml::MakeRefCounted<LayerTreePipeline>(
-          task_runners.GetPlatformTaskRunner() ==
-                  task_runners.GetRasterTaskRunner()
-              ? 1
-              : 2)),
-#endif  // FLUTTER_SHELL_ENABLE_METAL
+      layer_tree_holder_(std::make_shared<LayerTreeHolder>()),
```

Comment:
Based on a conversation with Chinmay:

> In terms of usage, there is no difference. The FML variant is intrusively ref-counted while the std variant is obviously not. There were arguments made when I first joined the team that extolled the virtues of the fml variant. While true, I never thought the benefits were significant enough for us to go invent/maintain our own. New code has always used std but the older subsystems have stuck to fml.
> Based on my vague recollection, the arguments were that the sizeof the std variant was massive relative to the size of a pointer and that the intrusive nature of ref counting would reduce the number of allocations necessary (this is not an issue if you use std::make_shared btw).
> I should go ping the folks who liked the FML variant for more arguments for it because I am very obviously biased against them and the utility they have provided.

So, it seems like using the std variant should be fine here.